db/seeds.rb

```diff
@@ -6,3 +6,8 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 Compliment.find_or_create_by(message: "You rock")
+Organization.find_or_create_by(
+  name: 'RailsBridge',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.